sklearn/ensemble/_hist_gradient_boosting/predictor.py

```diff
@@ -84,3 +107,20 @@ def compute_partial_dependence(self, grid, target_features, out):
             point.
         """
         _compute_partial_dependence(self.nodes, grid, target_features, out)
+
+
+def _make_known_categories(bin_thresholds, is_categorical):
```

Comment:
The fact that both these arguments come from the bin_mapper suggest that this should instead be a method of `_BinMapper`.